Packs/AggregatedScripts/Scripts/QuarantineFile/QuarantineFile.py

```diff
@@ -0,0 +1,1145 @@
+
+        self.pending_jobs = remaining_jobs
+
+    def _get_final_results(self) -> PollResult:
+        """
+        Formats and returns the final report after all jobs are complete.
+
+        This method cleans up the working data from the incident context, builds
+        a markdown table for the war room, and constructs the final CommandResults object.
+
+        Returns:
+            PollResult: A PollResult object with `continue_to_poll=False` and the final results.
+        """
+        demisto.debug("[Orchestrator] Formatting final results.")
+        # Clean up the context keys before returning the final result
+        demisto.debug("[Orchestrator] Cleaning up context keys.")
+        demisto.executeCommand(
+            "DeleteContext",
+            {"key": f"{QuarantineOrchestrator.CONTEXT_PENDING_JOBS},{QuarantineOrchestrator.CONTEXT_COMPLETED_RESULTS}"},
+        )
+
+        results_list = QuarantineResult.to_simple_list(self.completed_results)
+
+        # Build final report
+        final_readable_output = tableToMarkdown(
+            name=f"Quarantine File Results for: {self.args.get(self.FILE_PATH_ARG)}",
+            headers=["endpoint_id", "status", "message", "brand"],
+            t=results_list,
+            removeNull=True,
+        )
+
+        final_command_results = CommandResults(
+            outputs_prefix="QuarantineFile",
+            outputs_key_field=["endpoint_id", "file_path", "file_hash"],
+            readable_output=final_readable_output,
+            outputs=results_list,
+        )
+
+        # Prepend verbose results if the flag is set
+        if self.verbose:
+            self.verbose_results.append(final_command_results)
+            return PollResult(response=self.verbose_results, continue_to_poll=False)
+
+        demisto.debug("[Orchestrator] Final results report created.")
+        return PollResult(response=final_command_results, continue_to_poll=False)
+
+
+""" SCRIPT ENTRYPOINT """
+
+
+@polling_function(name="quarantine-file", timeout=arg_to_number(demisto.args().get("timeout", DEFAULT_TIMEOUT)))
+def quarantine_file_script(args: dict) -> PollResult:
+    """
+    Main polling script function that delegates all work to the Orchestrator.
+
+    This function is decorated with `@polling_function`, making it the entry point
+    for XSOAR's polling mechanism.
+
+    Args:
+        args (dict): The arguments for the script execution.
+
+    Returns:
+        PollResult: The result from the orchestrator's run.
+    """
+    if not args:
+        args = demisto.args()
+
+    orchestrator = QuarantineOrchestrator(args)
+    return orchestrator.run()
+
+
+def main():
+    """
+    Main execution block of the script.
+
+    It sets up the arguments, calls the main polling function, and handles
+    any top-level exceptions, returning an error to the user if one occurs.
+    """
+    demisto.debug(f"--- quarantine-file script started with arguments: {demisto.args()} ---")
```

Comment:
```suggestion
    demisto.debug(f"Command being called is quarantine-file,  with arguments: {demisto.args()} ---")
```
this is the message people tend to look for - its semi-standard